spec/support/capybara_webkit.rb

```diff
@@ -1,5 +1,3 @@
 Capybara.javascript_driver = :webkit
 
-Capybara::Webkit.configure do |config|
-  config.block_unknown_urls
-end
+Capybara::Webkit.configure(&:block_unknown_urls)
```

Comment:
A question: what was the violation Rubocop raised here?

Reply:
I honestly don't remember sorry

Reply:
Sometimes we can consciously ignore Rubocop -- as long as we have a reason. Side point  :)

Reply:
Side note, would you use rubocop with -a and get auto corrects or do them all manually?

Reply:
Depends how many changes are needed.
But even if rubicon was used with `-a` flag, I would use `git add -p` to confirm the changes I want to add interactively.